Day-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
 import './Button.css';
+
+function NumberButton(props){
+    return(
+        <button className={props.btn.buttonStyle}>{props.btn.text}</button>
```

Comment:
This looks identical to your `ActionButton` component. Why not have only one component?